### ISSUE TITLE
cant update number or checkbox fields

### ISSUE THREAD

**Issue by mrpn** [DavidZadrazil/airtable-api-php#1]
```
$response = $request->updateRecord(''. $urlid .'', ['Cancel' => '5']);
    $response->isSuccess();
```

tried to use example code to update Cancel field, it returns this error; 

How can I update non text fields? 

`Fatal error: Uncaught GuzzleHttp\Exception\ClientException: Client error: `PATCH https://api.airtable.com/v0/appXXXXXXXX/Invoices/recSMXnTwTiRrRvGB` resulted in a `422 Unprocessable Entity` response: {"error":{"type":"INVALID_VALUE_FOR_COLUMN","message":"Field Cancel can not accept value 5"}} in /var/www/airfinance.com/html/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:113 Stack trace: #0 /var/www/airfinance.com/html/vendor/guzzlehttp/guzzle/src/Middleware.php(65): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response)) #1 /var/www/airfinance.com/html/vendor/guzzlehttp/promises/src/Promise.php(203): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response)) #2 /var/www/airfinance.com/html/vendor/guzzlehttp/promises/src/Promise.php(156): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), Array) #3 /var/www/airfinance.com/html/vendor/guzzlehttp/promises/src/TaskQueue.php(47): GuzzleHttp\Promise\Promise: in /var/www/airfinance.com/html/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php on line 113`

**Comment by DavidZadrazil** [comment]
Hi,

what type of field are you trying to update?

**Comment by mrpn** [comment]
I tried to update Checkbox and Numeric value fields. Both give same error.

**Comment by DavidZadrazil** [comment]
I figured it was a bug. Airtable made changes to the API. I'm working on a fix.

**Comment by mrpn** [comment]
Thanks a lot 👍

**Comment by DavidZadrazil** [comment]
Ok. I've released [v0.0.3](https://github.com/DavidZadrazil/airtable-api-php/releases/tag/0.0.3) which fixes bug with `PATCH` requests. It should be available through Composer or you can download it [here](https://github.com/DavidZadrazil/airtable-api-php/archive/0.0.3.zip).

Tested and working, checkbox example update:
```php
$request->updateRecord('rec06ulkLhftmiLPP', ['SomeCheckboxField' => false]);
```
